articles/rules-are-dangerous.md

```diff
@@ -2,45 +2,45 @@
 authorName: Rolf Bruderer
 authorGithubUsername: bruderol
 issue: 31
-title: Rules are Dangerous
+title: Rules are dangerous
 ---
 # {{page.title}}
 
-Rules followed blindly in a dogmatic way do a lot of harm. This is an uncomfortable truth especially for us as engineers. As such we are used to see the world in rigid rules, true and false, if and else. We like to have crystal clear and exact definitions, guidelines or best practices on how to do something. This is not bad in the first place. But we all must be very careful that we do not fall into the trap of just following such rules blindly without thinking well about consequences. 
+Rules followed blindly in a dogmatic way do a lot of harm. This is an uncomfortable truth especially for us, as engineers. As such we are used to looking at the world in terms of rigid rules, true and false, if and otherwise. We like to have crystal clear and exact definitions, guidelines or best practices on how to do something. This is not a bad thing in itself. But we all must be very careful that we do not fall into the trap of just following such rules blindly without giving careful thought to consequences. 
 
-## Examples of Rules from Software Projects
+## Examples of rules from software projects
 
-Here are some examples of rules in Software Engineering that I have seen to cause troubles in projects:
+Here are some examples of rules in software engineering that I know from experience can cause troubles in projects:
```

Comment:
The correction sounds too harsh - i would never say it like so - this sounds as if this rules always cause troubles ... and this is not about a fishy thing as "my experience" but about "what I have seen in projects" - so concrete observations - therefore I like my old formulation better. I do not like the formulation "I know" - I didnt want to say that I know this - i wanted to say that i have seen them to cause troubles. To me thus is not the same.

Reply:
ok

Reply:
https://github.com/Zuehlke/fifty-shades/commit/36b464815149809af537129d854954e1e795eb1d